### DISCUSSION TITLE
Question about Android app named "FluidSynth"

### DISCUSSION THREAD

**Discussion by dfkettle** [FluidSynth/fluidsynth#1098] [Q&A] [answered]
Does anyone know if this Android app is an implementation of FluidSynth, or did they just steal the name? If it is an actual implementation, have they violated the license agreement?

https://play.google.com/store/apps/details?id=net.volcanomobile.fluidsynthmidi&hl=en_CA&gl=US

Thanks.

**Comment by derselbst** [comment, marked as answer]
As stated in the description of this app, it's a fork of fluidsynth. The source code is provided as well, I see no violation here.

**Reply by dfkettle** [reply]
> As stated in the description of this app, it's a fork of fluidsynth. The source code is provided as well, I see no violation here.

I meant the app in the Google PlayStore (the link above), from a company called "Volcano Mobile (https://volcanomobile.net/). I wasn't sure if it was the same, or if they just used the same name. The reason I ask is that I'm trying to get SCAMP (http://scamp.marcevanstein.com/) running on my Android tablet, but one of the requirements is FluidSynth. I installed the app from the PlayStore (not from here), but it doesn't seem to work.

I sent them an email asking if it was the same thing, but I haven't recevied any response yet.

Sorry if I caused confusion.

**Reply by derselbst** [reply]
> I meant the app in the Google PlayStore (the link above)

I meant this app as well. I advised you to read the description of this app. A link to the source code is provided as well. A fork of fluidsynth is not the same as upstream fluidsynth.

> The reason I ask is that I'm trying to get SCAMP running on my Android tablet

I don't think it will be so simple by just installing a random app called "fluidsynth" from the playstore and it will work. You'll have to dig deeper.

**Comment by ghost** [comment]
The docs was designed to be followed in a computer so some of the steps presented is unavailable in Android. If you insists, you could try installing Termux android application and there, install python, then `pip install scamp && apt install fluidsynth lilypond`. Though I have not tested this. You might be the firsts person to test this out.
![Screenshot_20220618-200654_Termux](https://user-images.githubusercontent.com/46589135/174436784-478c0178-2d02-4ec6-a828-aae8f13ea45d.jpg)

**Reply by dfkettle** [reply]
Thanks, I tried that already. I was able to install scamp and scamp-extensions, but fluidsynth wasn't found in the default repositories ("Unable to locate package fluidsynth"). If I knew which repository it was in, I might be able to add it, though. I don't really care about lilypond, I don't need to print scores from my tablet.

**Reply by ghost** [reply]
> I don't really care about lilypond, I don't need to print scores from my tablet.

What were you trying to accomplish rather than printing scores? Composing music with a text editor perhaps? :)

P.S Fluidsynth exist on Android > 6.0.0. If you're running on > 6.0.0 and it still cannot find fluidsynth, try updating with `apt update`.
